internal/testhelpers/testhelpers.go

```diff
@@ -27,10 +28,50 @@ func EnsureBuffaloCMD(t *testing.T) error {
 		return fmt.Errorf("not in the cli source folder")
 	}
 
-	ex := exec.Command("go", "install", "-tags", "sqlite", "github.com/gobuffalo/cli/cmd/buffalo")
+	ex := exec.Command("go")
```

Comment:
Is this built binary also cleaned up at some point after the test?

Reply:
no, it's not, actually one of the ideas I was having was to reuse existing compiled buffalo binary.